sdap_ingest_manager/ingestion_order_store/FileIngestionOrderStore.py

```diff
@@ -0,0 +1,9 @@
+from sdap_ingest_manager.ingestion_order_store import IngestionOrderStore
```

Comment:
We can take some time tomorrow morning to discuss that. I am not getting how you foresee this IngestionOrderStores to work.